### ISSUE TITLE
az storage container generate-sas --ip does not accept IP_RANGE

### ISSUE THREAD

**Issue by ezYakaEagle442** [Azure/azure-cli#13823]
[https://docs.microsoft.com/en-us/cli/azure/storage/container?view=azure-cli-latest#az-storage-container-generate-sas](https://docs.microsoft.com/en-us/cli/azure/storage/container?view=azure-cli-latest#az-storage-container-generate-sas)

```sh
az version

AUTHORIZED_IP_RANGE="176.134.171.0/24"

az storage container generate-sas \
    --account-name $aro_registry_blob_str_name \
    --name $aro_registry_container_name \
    --account-key $str_acc_access_key1 \
    --permissions acdlrw \
    --expiry "2020-06-10T17:00Z" \
    --auth-mode login \
    --https-only \
    --ip $AUTHORIZED_IP_RANGE
```

The above command returns the error below :
az storage container generate-sas: error: argument --ip: invalid ipv4_range_type value: '176.134.171.0/24'

{
  "azure-cli": "2.7.0",
  "azure-cli-command-modules-nspkg": "2.0.3",
  "azure-cli-core": "2.7.0",
  "azure-cli-nspkg": "3.0.4",
  "azure-cli-telemetry": "1.0.4",
  "extensions": {
    "aks-preview": "0.4.45",
    "aro": "1.0.0",
    "azure-firewall": "0.3.0",
    "connectedk8s": "0.1.3",
    "k8sconfiguration": "0.1.6"
  }
}

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: adcba5ed-e693-6345-2c1b-c17422d7b8f9
* Version Independent ID: 58297400-b777-6868-1687-0af3a512a16f
* Content: [az storage container](https://docs.microsoft.com/en-us/cli/azure/storage/container?view=azure-cli-latest#az-storage-container-generate-sas)
* Content Source: [src/azure-cli/azure/cli/command_modules/storage/_help.py](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/storage/_help.py)
* Service: **storage**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by ezYakaEagle442** [comment]
it looks like AUTHORIZED_IP_RANGE="176.134.171.0-176.134.171.255" is accepted

**Comment by yonzhan** [comment]
add to S172

**Comment by Juliehzl** [comment]
> it looks like AUTHORIZED_IP_RANGE="176.134.171.0-176.134.171.255" is accepted

You are right for the usage and its current supported ip range type. `-` should be used to specify the ip range. 
And we cannot change current argument value for `--ip` because it will cause breaking changes to users. Sorry for inconvenience. 
And I  will close the issue and if you still have problem, feel free to reopen it and find me for help.

**Comment by Juliehzl** [comment]
We will refine the error message for easy usage.
